t/10-tests_overview.t

```diff
@@ -233,11 +233,40 @@ $t->get_ok('/tests/overview?distri=opensuse&version=13.1&groupid=1001&groupid=10
 $summary = get_summary;
 like(
     $summary,
-    qr/Summary of opensuse, opensuse test/i,
-    'multiple groups with no build specified yield latest build of first group'
+    qr/Summary of opensuse, opensuse test build 0091 /i,
+    'multiple groups with no build specified yield latest build of every group'
 );
 like($summary, qr/Passed: 2 Failed: 0 Scheduled: 1 Running: 2 None: 1/i);
 
+my $jobGroup = $t->app->schema->resultset('JobGroups')->create(
+    {
+        id         => 1003,
+        sort_order => 0,
+        name       => 'opensuse test 2'
+    });
+
+my $job = $t->app->schema->resultset('Jobs')->create(
+    {
+        id       => 99964,
+        BUILD    => '0092',
+        group_id => 1003,
+        TEST     => 'kde',
+        DISTRI   => 'opensuse',
+        VERSION  => '13.1'
+    });
+
+$t->get_ok('/tests/overview?distri=opensuse&version=13.1&groupid=1001&groupid=1003')->status_is(200);
+$summary = get_summary;
+like(
+    $summary,
+    qr/Summary of opensuse, opensuse test 2 build 0091,0092 /i,
+    'multiple groups with no build specified yield latest build of every group'
```

Comment:
The test label is now the same as the previous one.
Labels should be different for each test, so that it is easy to find out which test failed.

Reply:
incorporated into https://github.com/os-autoinst/openQA/pull/3973